packages/cli/commands/executeAll.ts

```diff
@@ -172,7 +207,13 @@ export class ExecuteAll extends Command {
 				}
 
 				const serializedData = JSON.stringify(data, null, 2);
-				GenericHelpers.logOutput(`Workflow ${workflowData.id} succeeded.`);
+				if (json === true) {
+					if (flags.compare === undefined){
+						GenericHelpers.logOutput(JSON.stringify(executionResult, null, 2));
```

Comment:
Also the comparison result should be part of the resulting JSON object, so it also outlines what changed.

Reply:
The comparison result (changes) is included with the execution result for each workflow